lib/OpenQA/Schema/Result/Users.pm

```diff
@@ -51,6 +51,14 @@ __PACKAGE__->add_columns(
         false_id      => ['0', '-1'],
         default_value => '0',
     },
+    last_login_version => {
+        data_type   => 'text',
+        is_nullable => 1,
+    },
+    feature_informed => {
```

Comment:
@foursixnine It is for checking if the user is already informed about the latest/current feature version or not. If he saw the newest features the boolean is true.